src/tests/end-to-end/common/element-identifiers/common-selectors.ts

```diff
@@ -11,3 +11,16 @@ export const CommonSelectors = {
 export const GuidanceContentSelectors = {
     mainContentContainer: '.content-container',
 };
+
+export const fastPassSelectors = {
+    tabStopNavButtonSelector: 'a[title="Tab stops"]',
+    tabStopToggleCheckedSelector: 'button[aria-label="Tab stops"][aria-checked=true]',
+    tabStopToggleUncheckedSelector: 'button[aria-label="Tab stops"][aria-checked=false]',
+    tabStopVisulizationStart: '.insights-tab-stops',
+};
+
+export const targetPageSelectors = {
+    targetPageNativeWidgetIFrameName: 'native-widgets',
+    targetPageNativeWidgetFirstRadio: '#input-radio-1',
+    targetPageNativeWidgetH1: '#h1-native-widget',
```

Comment:
I added a `target-page-selectors.ts` file in my PR (#993) that might be a more relevant place for these selectors.

Reply:
Moved the selectors that I was using to `target-page-selectors.ts`